geopy/geocoders/opencage.py

```diff
@@ -116,8 +116,8 @@ def geocode(
         :param str country: Provides the geocoder with a hint to the
```

Comment:
According to the docs at https://opencagedata.com/api#forward-opt, the `countrycode` param *restricts* the results to the specified country, however the current description there says that it just provides a *hint*. Could you please update this description?

Reply:
Good point! I changed that.